pages/docs/support/config-files.mdx

```diff
@@ -0,0 +1,672 @@
+---
+title: "Configuration Files"
+date: "2022-02-09"
+pullQuote: "Definitions and examples of Grouparoo configuration files."
+---
+
+Grouparoo projects contain configuration files for Apps, Models, Sources, Properties, Groups, Destinations, and any other objects is exported into config files.
+These config files can be created and updated with [UI config](/docs/config).
+
+Config files are either JSON or JavaScript which allow you the flexibility to manipulate data or load configuration from external sources.
```

Comment:
I left references to the `JavaScript` files from code config. But should this be removed? In a world without generators should we only encourage JSON files?